src/main/java/xyz/groundx/caver_ext_kas/kas/wallet/Wallet.java

```diff
@@ -107,6 +106,11 @@
      */
     RPC rpc;
 
+    /**
+     * The endpoint url set in RPC.
+     */
+    String endpointUrl;
```

Comment:
What does mean endpointUrl? Wallet API's url? or Node API's url?

Reply:
I changed variable name to rpcUrl.